pkg/cmd/cli/cmd/importimage.go

```diff
@@ -26,12 +26,21 @@ import (
 
 var (
 	importImageLong = templates.LongDesc(`
-		Import tag and image information from an external Docker image repository
+		Import the latest images from a Docker registry into an image stream
```

Comment:
I know you mention this explicitly at the end of this description, but users usually read first sentence at most and look back only when in doubt.